### DISCUSSION TITLE
Interested in contributing

### DISCUSSION THREAD

**Discussion by jherren** [davidm-glitch/home-assistant-eheim-digital#16] [General]
I am currently setting up my Eheim and would like to contribute how I can to the project. Below is the list of equipment I have. Let me know if there is anything I can do to assist or where to start contributing.

Eheim thermocontrol+ 250W
Eheim powerLED+ fresh daylight 26W
Eheim powerLED+ fresh plants 29.8W
Eheim powerLED+ power supply 63-80W
Eheim LEDcontrol+ wireless LED controller
Eheim professionel 5e 450

**Comment by jkhalil** [comment]
@jherren That would be fantastic. I'm trying to work with another individual who started to create this as a HA core integration as outlined in  issue #13 and waiting to hear back. 

The current integration was really a quick and dirty build over 2 weeks as I tried to help @davidm-glitch do it. In it's current form, the integration is missing a lot of current features and updated HA feature sets and has a nasty websocket issue. Hence the rewrite.
